File: sub-web-model/src/main/java/org/github/base/mapper/ISysUserRoleMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysUserRoleEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 用户与角色对应关系 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-07
 */
@MyBatisMapper
public interface ISysUserRoleMapper extends IMapper<SysUserRoleEntity> {}

File: sub-mysql-sccore-service/src/main/java/org/github/mysql/sccore/base/mapper/ISysUserRoleMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.sccore.base.mapper;

import org.github.mysql.sccore.base.entity.SysUserRoleEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 用户角色管理 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ISysUserRoleMapper : IMapper<SysUserRoleEntity>
